Tutorials/start/configuring-faraday.md

```diff
@@ -12,7 +12,7 @@ The application files are located:
 
 ## Device Configuration Setup
 
-`Deviceconfiguration.py` uses two .ini files, one to talk to Proxy and one to program Faraday hardware with. First we should ensure the program can communicate with Proxy running in the background. For this we need to edit the `[DEVICES]` section of `deviceconfiguration.ini` to contain the same configuration Proxy is configured with in `proxy.ini`
+First we should ensure the Device Configuration program can communicate with Proxy running in the background. For this we need to edit the `[DEVICES]` section of `deviceconfiguration.ini` to contain the same configuration Proxy is configured with in `proxy.ini`
 
 `[FLASK]`: Flask server configuration values
```

Comment:
Not sure what you're talking about @kb1lqd? This text doesn't comment on multiple units and `deviceconfiguration.ini` only contains configuration for a single unit.